azerothcore/azerothcore-wotlk
# feat(Core/Authserver): install authserver as system service on Windows

Install AuthServer As System Service on Windows

## Changes Proposed:
-  Install AuthServer As System Service on Windows


## Issues Addressed:
- Closes #4223



## Tests Performed:
- Build on Windows 10 20H2 with CMake 3.19.1 and Visual Studio 2019.
- Assembly database with git-bash and db_assembler.sh.
- Run cmd.exe as admin.
- Exec: authserver.exe -s install
- Go to windows services and search AzerothCore auth service.
- Start the service and see the Auth.log file.



## How to Test the Changes:



## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
